src/Components/DashBoard/Repository/RepoComponents/RepoDetails/RepoLeftPaneComponent.js

```diff
@@ -62,6 +62,24 @@ export default function RepoLeftPaneComponent(props) {
     return remoteLogo;
   };
 
+  const remoteUrl = () => {
+    let remoteData = "";
+    if (gitRemoteData) {
+      if (gitRemoteData.match(/(^https)/gi)) {
```

Comment:
Add this regex `(^https://)` for matching as the current one will match invalid URLs as well (E.g. : httpsgithub.com/neel1996/gitconvex.git)